src/cephadm/cephadm

```diff
@@ -505,7 +507,7 @@ class Monitoring(object):
         'prometheus': {
             'image': DEFAULT_PROMETHEUS_IMAGE,
             'cpus': '2',
-            'memory': '4GB',
+            'memory': '8GB',
```

Comment:
what's this based on? Are we seeing issues with prometheus and need to up its memory limit to fix it? 8GB is more than most daemons get.

Reply:
Sorry this change needs to be reverted, will fix it!